pkg/pulumiyaml/codegen/convert.go

```diff
@@ -49,7 +49,17 @@ func ConvertTemplateIL(template *ast.TemplateDecl, loader schema.ReferenceLoader
 		loader = schema.NewPluginLoader(host)
 	}
 
-	templateBody, tdiags := ImportTemplate(template, pulumiyaml.NewPackageLoaderFromSchemaLoader(loader))
+	pkgLoader := pulumiyaml.NewPackageLoaderFromSchemaLoader(loader)
+	_, tdiags, err := pulumiyaml.PrepareTemplate(template, pkgLoader)
```

Comment:
Might help to comment why it's OK to discard the runner return value here - I think it's because (a.) we aren't going to execute the template and (b.) the loader stored on the runner is already in our local scope and reused?